shared/helpers/cache.py

```diff
@@ -269,3 +269,6 @@ async def wrapped(*args, **kwargs):
             return result
 
         return wrapped
+
+
+cache = OurOwnCache()
```

Comment:
I actually think you should put this in a file in the worker... but this is a personal and nit opinion.

Reply:
Oh yeah, I guess I can leave the instance in `worker` and `codecov-api` and then just wrap the `shared` function there (but with caching)